### DISCUSSION TITLE
Allow not only indexName but also full url (when working with reverse proxy)

### DISCUSSION THREAD

**Discussion by xorock** [algolia/instantsearch#5540] [Ideas]
Hi.
Although the indexName value fits perfectly with the idea of meilisearch indexes it is not quite safe. The prepared URL requires opening access directly to http://ms:7700/. It is much safer to prepare the address under a server's reverse proxy that contains only the needed index.

```bash
location /movies-index {
        [...]
        proxy_pass  http://ms:7700/indexes/movies/search/;
    }
```

What do you think about the proposal to make the indexName value take the name of the index or the full URL. Or the second option - introduce a new value and one of the two must be required.

**Comment by Haroenv** [comment]
This is an issue for https://github.com/meilisearch/instant-meilisearch
